src/index.js

```diff
@@ -18,7 +23,159 @@ import { months } from './data.js'
   leftArrow.addEventListener('click', () => { scrollItems('left') })
   rightArrow.addEventListener('click', () => { scrollItems('right') })
   document.getElementById('select-icon').addEventListener('change', updateMiddleCitiesScrollButtonVisibility)
+  changeBottomByContinents('down-arrow')
+  changeBottomByTemperature('up-arrow-temp')
+  document.querySelector('.continent-arrow').addEventListener('click', (e) => { changeBottomByContinents(e.target.id) })
+  document.querySelector('.temperature').addEventListener('click', (e) => { changeBottomByTemperature(e.target.id) })
+
+  /**
+   *  This function will update the bottom container elements based on continent sorting criteria
+   * @param {string} arrowId It is the arrow id of the arrow image
+   */
+  function changeBottomByContinents (arrowId) {
+    const continentArrow = document.getElementsByClassName('continent-arrow')[0]
+    if (arrowId === 'down-arrow') {
+      bottomItemCount = 0
+      continentArrow.src = '/Assets/HTML&CSS/GeneralImages&Icons/' + 'arrowUp' + '.svg'
```

Comment:
Can the arrow svg files names as per the arrowId argument we're getting? `down-arrow` and `up-arrow`. This will help us having one single assignment with the argument variable we're receiving

Reply:
If we're altering the arrow as per design can use ternary operator and still compute in one expression.
eg. continentArrow.src = `/Assets/HTML&CSS/GeneralImages&Icons/${arrowId === 'down-arrow' ? 'up-arrow' : 'down-arrow'}svg`